test/cmd/basicresources.sh

```diff
@@ -263,6 +263,14 @@ os::cmd::expect_success_and_text "oc get route route-with-set-port --template='{
 echo "expose: ok"
 os::test::junit::declare_suite_end
 
+# Test that resource printer includes resource kind on multiple resources
+os::test::junit::declare_suite_start "cmd/basicresources/get"
+os::cmd::expect_success 'oc create imagestream test1'
+os::cmd::expect_success 'oc new-app node'
+os::cmd::expect_success_and_text 'oc get all' 'is/test1'
+echo "resource printer: ok"
+os::test::junit::declare_suite_end
```

Comment:
:+1: @juanvallejo this works for me for validating that the kind gets automatically added.

Of course there could still be other kinds that will print without the prefix, but that's a limitation we seen to be accepting. For example, printing a new kind will require adding some code to the printer of that kind.

As for this test here and now, @juanvallejo do you think it would make sense to check that `oc get is` does not include the `is/` prefix, while `oc get all` does?